lib/service/Service.ts

```diff
@@ -1,13 +1,14 @@
 import Logger from '../Logger';
 import Pool from '../p2p/Pool';
 import OrderBook, { OrderArrays } from '../orderbook/OrderBook';
-import LndClient from '../lndclient/LndClient';
+import LndClient, { LndClientConfig } from '../lndclient/LndClient';
```

Comment:
I actually think we should be able to drop config information from this class given the distinction between an lnd client being "disabled" and "disconnected", see comments below.

Reply:
done